### ISSUE TITLE
Checkstyle requires inconsistent indentation for lambda arguments

### ISSUE THREAD

**Issue by lbenedetto** [checkstyle/checkstyle#13539]
```bash
/var/tmp $ javac TestEnum.java
/var/tmp $ cat config.xml
```
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN" "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <module name="Indentation"/>
  </module>
</module>
```
```bash
/var/tmp $ cat TestEnum.java
```
```java
import java.util.function.Consumer;
import java.util.function.Function;

public enum TestEnum {
  NAME(
      String::getClass,
      any -> {}
  );


    TestEnum(
        Function<String, Class<?>> function,
        Consumer<String> consumer
    ) {

    }
}
```
```bash
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-10.12.2-all.jar -c config.xml TestEnum.java
Starting audit...
[ERROR] /var/tmp/TestEnum.java:9:7: 'lambda arguments' has incorrect indentation level 6, expected level should be 10. [Indentation]
Audit done.
Checkstyle ends with 1 errors.
```
---

**Describe what you expect in detail.**
This should be valid, but is not
```java
  NAME(
      String::getClass,
      any -> {}
  );
```
This should not be valid, but it is
```java
  NAME(
      String::getClass,
          any -> {}
  );
```

**Comment by nrmancuso** [comment]
Possible duplicate of https://github.com/checkstyle/checkstyle/issues/11103

**Comment by lbenedetto** [comment]
Actually, that issue seems to have been fixed at some point. If I make the requested change (indentation level 10 instead of 6) then checkstyle reports no errors.

**Comment by romani** [comment]
fixed in scope of https://github.com/checkstyle/checkstyle/issues/11103
